server/router/api.go

```diff
@@ -144,6 +163,8 @@ func apiRoutes(e *gin.Engine) {
 		queue.GET("/info", api.GetQueueInfo)
 		queue.GET("/pause", api.PauseQueue)
 		queue.GET("/resume", api.ResumeQueue)
+		queue.GET("/norunningpipelines", api.BlockTilQueueHasRunningItem)
+		// DEPRECATED - use /norunningpipelines
 		queue.GET("/norunningbuilds", api.BlockTilQueueHasRunningItem)
```

Comment:
you did breake api in other places ... just break i there too

Reply:
yes and since we require same version for server,agent and cli ... we **can** break it

Reply:
I'll send a seperate pull